### DISCUSSION TITLE
Why is content in subfolders not show on website?

### DISCUSSION THREAD

**Discussion by Kajakster** [datenstrom/community#647] [Ask a question] [answered]
Again I am working with a fresh Install out of the Box. 

Content in Subfolders does not appear on the Website.  I am aware subfolders are not allowed in the  /home directory but also in other folders a subfolder does not work.
![subfolders](https://user-images.githubusercontent.com/83357904/116449026-063b3a80-a85a-11eb-86e8-d39ad6eb1eac.JPG)

Content in Folder 31-subpage does not appear on the website, also not in the Menu. 

Maybe I have a serious misunderstanding, but I can't find help in the documentation. 
Has anyone a Idea what to look after?

**Comment by nzyeehaa** [comment, marked as answer]
it looks like you are doing everything correctly...

I have assumed you have a page.md inside your subpage folder...
...and you can test it by manually typing your url; i.e: site/Test/subpage/ - and your page.md should appear...

The part you need to know is **navigation-tree** was removed some time ago and default templates only have a **single layer** menu. So unfortunately, you now need to get familiar with the API and "roll your own" layout to have the menu reflect your structure.

Ref: 
https://github.com/datenstrom/yellow/discussions/462#discussioncomment-244854
https://github.com/datenstrom/yellow/discussions/507#discussioncomment-244865

**Reply by Kajakster** [reply]
Thanks for pointing that out.

Pages are accessible by typing the URL.

What puzzled me Is the removed *NavigationTree* 

Maybe this could be pointed out in the documentation where content in subfolders is explained.

**Reply by nzyeehaa** [reply]
the code/API has changed over time, but this should work from version 0.8.13...
(would need customising to your own tastes for html layout and css)

navigation-tree.html
```
<?php list($name, $pages, $level) = $this->yellow->getLayoutArguments() ?>
<?php if (!$pages) $pages = $this->yellow->content->top() ?>
<?php $this->yellow->page->setLastModified($pages->getModified()) ?>
<ul>
<?php foreach ($pages as $page): ?>
<?php $children = $page->getChildren() ?>
 <li><a<?php echo $page->isActive() ? " class=\"active\" aria-current=\"page\"" : "" ?> href="<?php echo $page->getLocation(true) ?>"><?php echo $page->getHtml("titleNavigation") ?></a><?php if ($children->count()) { echo "\n "; $this->yellow->layout($name, $children, $level+1); } ?></li>
<?php endforeach ?>
</ul>
```

**Reply by Kajakster** [reply]
Thanks for the code.

I just thought of another way to handle this.

Install the [toc] extension and then put all the Content on one page. No Subfolders. 

It is a whole different approach, but I somehow like the idea. Users can jump on the headline / topic they are interested in and then scroll down or up to explore the rest.

**Reply by nzyeehaa** [reply]
would kill SEO and search engine rank
(and wear out the scroll wheel + an otherwise useful finger)

**Comment by nzyeehaa** [comment]
p.s: looks like [Paris theme](https://github.com/datenstrom/yellow-extensions/tree/master/source/paris) caters for child menu items

**Reply by Kajakster** [reply]
yes, thank you so much!
